src/extension/content-script/batteries/Reddit.ts

```diff
@@ -0,0 +1,100 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+declare global {
+  interface Window {
+    LBE_REDDIT_MUTATION_OBSERVER: MutationObserver;
+  }
+}
+
+const urlMatcher = /^https:\/\/www.reddit\.com\/user\/(\w+).*/;
+
+// can we get the user description from the primary column that holds the profile information?
+function isOnProfilePage() {
+  return document.querySelector('[class="bVfceI5F_twrnRcVO1328"]') != null;
```

Comment:
Nice PR! We had a look yesterday at this and didn't trust those classes. Can we be sure these won't change? They do look dynamic. That's why we were worried a bit.  
What do you think?